scripts/eurostat/health_determinants/common/dcid_existance.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""_summary_
+Script to check the property/dcid/nodes existance in datacommons.org.
+"""
+import datacommons
+
+
+def check_dcid_existance(nodes: list) -> dict:
+    """
+    Checks the existance of dcid nodes in autopush.
+    True: Available
+    False: Unavailable
+    Args:
+        nodes (list): Dcid Nodes List
+
+    Returns:
+        dict: Status dictionary.
+    """
+    # pylint: disable=protected-access
+    nodes_response = datacommons.get_property_values(
+        nodes,
+        "typeOf",
+        out=True,
+        value_type=None,
+        limit=datacommons.utils._MAX_LIMIT)
```

Comment:
if nodes list is longer than _MAX_LIMIT, can do batches of upto that size.

Reply:
The number of nodes are less than 100 hence, we have used _MAX_LIMIT.